modules/uncertainty/R/run.ensemble.analysis.R

```diff
@@ -108,12 +108,12 @@ run.ensemble.analysis <- function(settings, plot.timeseries = NA, ensemble.id =
       
       hist(my.dat,xlab=units,
            main="",cex.axis=1.1,cex.lab=1.4,col="grey85")
-      box(lwd = 2.2)
+      graphics::box(lwd = 2.2)
       
       boxplot(my.dat,ylab=units,
               boxwex=0.6,col="grey85", cex.axis=1.1,range=2,
               pch=21,cex=1.4, bg="black",cex.lab=1.5)
-      box(lwd=2.2)
+      graphics::box(lwd=2.2)
```

Comment:
A trivial formatting fix, primarily because the CI checks seem stuck and applying this will prompt them to run again:

```suggestion
      graphics::box(lwd = 2.2)
```